midas-author/lps/src/app/app.module.ts

```diff
@@ -61,7 +62,8 @@ enableProdMode();
         GoogleAnalyticsService,
         DatePipe,
         UserMessageService,
-        // fakeBackendProvider
+        NgbActiveModal,
+        fakeBackendProvider
```

Comment:
I'm concerned about `fakeBackendProvider` here:  does this need to be commented out?